.travis.yml

```diff
@@ -306,6 +310,12 @@ before_install:
         # this assumes TEST_THEANO runs in the same build as TEST_MATPLOTLIB
         conda install theano;
     fi
+
+  - if [[ "${TEST_SYMENGINE}" == "true" ]]; then
+        conda config --add channels conda-forge;
+        conda config --add channels symengine;
+        conda install python-symengine=0.1.0.dev0+97c5a21
```

Comment:
Perfect, thanks! +1 to merge from me.

@asmeurer, would you mind reviewing it once more?

Sent from my mobile phone.
On Jul 24, 2016 11:13 PM, "Isuru Fernando" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/sympy/sympy/pull/11002#discussion_r72009526>:
>
> > @@ -306,6 +310,12 @@ before_install:
> >          # this assumes TEST_THEANO runs in the same build as TEST_MATPLOTLIB
> >          conda install theano;
> >      fi
> > +
> > +  - if [[ "${TEST_SYMENGINE}" == "true" ]]; then
> > +        conda config --add channels conda-forge;
> > +        conda config --add channels symengine;
> > +        conda install python-symengine=0.1.0.dev0+97c5a21
>
> @certik <https://github.com/certik>, pinned version 0.1.0.dev0+97c5a21
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sympy/sympy/pull/11002/files/d909aaf58e3ec60ba6acdd704489774df200b693#r72009526>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABQWJSNtuYZTophGfq813mU8vpDU6wlks5qZEYNgaJpZM4IF-EF>
> .
>